### PR TITLE
config: add Astra Linux support

### PR DESCRIPTION
**Pull request by foxzi**
Related to https://github.com/tarantool/infra/issues/190

### CODE REVIEW

**Comment by ylobankov** [comment]
Please update commit message to something like this:
```
config: add Astra Linux support

Related to tarantool/infra#190
```

File: config.default [hunk 1]
```diff
@@ -91,6 +91,12 @@
             "7.3"
           ]
         },
+        "astralinux": {
```
Comment (ylobankov) [inline config.default:94]: Please update changelog file.

File: config.default [hunk 2]
```diff
@@ -91,6 +91,12 @@
...
+          "base": deb,
```
Comment (ylobankov) [inline config.default:95]: This will break the config file. Please wrap `deb` into `"`.

File: config.default (hunk 1 above)
Comment (foxzi) [reply config.default:94]: done

File: config.default (hunk 2 above)
Comment (foxzi) [reply config.default]: fixed

**Comment by foxzi** [comment]
> Please update commit message to something like this:
> 
> ```
> config: add Astra Linux support
> 
> Related to tarantool/infra#190
> ```

done

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added support of Astralinux 1.7.
```
Comment (ylobankov) [inline CHANGELOG.md:12]: Astralinux -> Astra Linux

Comment (foxzi) [reply CHANGELOG.md]: done

**Review by ylobankov — APPROVED** [review]

**Pull request merged**